components/aws/sagemaker/train/src/sagemaker_training_component.py

```diff
@@ -64,6 +64,14 @@ def _get_job_status(self) -> SageMakerJobStatus:
                 error_message=message,
                 raw_status=status,
             )
+        if status == "Stopped":
+            message = "The job reached Stopped state"
+            return SageMakerJobStatus(
+                is_completed=True,
+                has_error=True,
+                error_message=message,
```

Comment:
is the error message always present for stopped jobs? Does this code work fine even if the message is not present?

Reply:
i manually add the `message` on line 68
ya `error_message` in optional